client/src/Components/Button_desktop.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+const STYLES = ['btn--primary', 'btn--outline'];
+
+const SIZES = ['btn--medium', 'btn--large'];
```

Comment:
This is an interesting approach! 
However, I think you can integrate this even better in styled components. 
You work in your other components with styled components so I would suggest you don't mix them with css classes to stay consistent

Reply:
Check, thx!